packages/core/types/event.d.ts

```diff
@@ -62,7 +62,7 @@ interface Stackframe {
   inProject?: boolean
 }
 
-interface Error {
+export interface Error {
```

Comment:
Exported so it can be used internally but not re-exported from index.d.ts so still not available publicly.